Message Board/View Controllers/MessageThreadsTableViewController.swift

```diff
@@ -13,6 +13,8 @@ class MessageThreadsTableViewController: UITableViewController {
     override func viewWillAppear(_ animated: Bool) {
         super.viewWillAppear(animated)
         
+        tableView.accessibilityLabel = "MessageThreadsTableView"
```

Comment:
Great work adding more accessiblity hints in your code.

1. Next time use the `accessibilityIdentifier`
2. You should do this in the `viewDidLoad()` method, it only needs to be set once